ophyd/pseudopos.py

```diff
@@ -126,11 +334,14 @@ class PseudoPositioner(Device, Positioner):
     '''
     def __init__(self, prefix, *, concurrent=True, read_attrs=None,
                  configuration_attrs=None, monitor_attrs=None, name=None,
-                 **kwargs):
+                 egu='', **kwargs):
```

Comment:
It makes sense on a `SoftPositioner` as well, everything needs to have a unit at some level.